content/search-github/searching-on-github/searching-issues-and-pull-requests.md

```diff
@@ -160,6 +160,15 @@ You can use the `involves` qualifier to find issues that in some way involve a c
 | <code>involves:<em>USERNAME</em></code> | **[involves:defunkt involves:jlord](https://github.com/search?q=involves%3Adefunkt+involves%3Ajlord&type=Issues)** matches issues either @defunkt or @jlord are involved in.
 | `in:body` <code>involves:<em>USERNAME</em></code> | [**NOT bootstrap in:body involves:mdo**](https://github.com/search?q=NOT+bootstrap+in%3Abody+involves%3Amdo&type=Issues) matches issues @mdo is involved in that do not contain the word "bootstrap" in the body.
 
+## Search for my issues and pull requests
+
+To quickly view your issues and pull requests, navigate to the respective links from your GitHub dashboard. More advanced search options are available with `@me`, either by using the global search bar, or by filtering results from an individual repository. Any qualifier that accepts a username will allow you to limit your search to issues and pull requests you have created or have interacted with. To do so, follow the desired qualifier with `@me`.
```

Comment:
```suggestion
You can search for issues and pull requests you have created or have interacted with by following the desired qualifier with `@me`. Any qualifier that works with a username allows you to limit your search to issues and pull requests you created, are assigned, mentioned on, or are requested as a reviewer of.
```

I think this intro can be a bit shorter. ✨

Reply:
Great suggestion @sophietheking ! This has a much better flow. Thanks so much for the review! 💥